src/pages/index.js

```diff
@@ -3,22 +3,173 @@ import "../pages/index.css";
+  nameSelector: ".profile__title",
+  jobSelector: ".profile__description",
+  avatarSelector: ".profile__image",
+});
+
+// ------------------------------Api -----------------------------------------
+const api = new Api({
+  baseUrl: "https://around.nomoreparties.co/v1/cohort-3-en",
+  headers: {
+    authorization: "ac57ec6d-a8c4-4053-9947-30120afdc027",
+    "Content-Type": "application/json",
+  },
+});
+
+// //-------------------------------- Card Function ----------------------------
+let currentUserId;
+let section;
+
+api
+  .getInfo()
+  .then(([userData, initialCards]) => {
+    currentUserId = userData._id;
+    userInfo.setUserInfo({ title: userData.name, job: userData.about });
+    userInfo.setAvatarInfo(userData.avatar);
+    section = new Section(
+      {
+        items: initialCards,
+        renderer: (data) => {
+          const newCard = createCard(data);
+          section.addItem(newCard);
+        },
+      },
+      cardListEl
+    );
+    section.renderItems();
+  })
+  .catch((err) => {
+    console.log(err);
+  });
+
+function createCard(cardData) {
+  const card = new Card({
+    data: cardData,
+    cardSelector: "#card-template",
+    previewImage: handleCardImageClick,
+    myUserId: currentUserId,
+    handleDeleteCard: handleCardDelete,
+    handleLikes: (isLiked) => {
+      if (!isLiked) {
+        api
+          .addLikes(cardData._id)
+          .then((data) => {
+            card.updateLikes(data.likes);
+          })
+          .catch((err) => {
+            console.log(err);
+          });
+      } else {
+        api
+          .removeLikes(cardData._id)
+          .then((data) => {
+            card.updateLikes(data.likes);
+          })
+          .catch((err) => {
+            console.log(err);
+          });
+      }
+    },
+  });
+
+  return card.generateCard();
+}
+//---------------------------------- Submit functions ---------------------------------
+function handleEditProfileFormSubmit({ title, description }) {
+  editProfilePopup.renderLoading(true);
+  api
+    .userInfoUpdate(title, description)
+    .then(() => {
+      userInfo.setUserInfo({ title, job: description });
+      editProfilePopup.close();
+    })
+    .catch((err) => {
+      console.error(err);
+    })
+    .finally(() => {
+      editProfilePopup.renderLoading(false);
+    });
+}
+function HandleNewCardSubmit({ title, link }) {
+  newCardPopup.renderLoading(true);
+  api
+    .addCard(title, link)
+    .then((cardData) => {
+      const newCard = createCard(cardData);
+      section.addItem(newCard);
+      newCardPopup.close();
+    })
+    .catch((err) => {
+      console.error(err);
+    })
+    .finally(() => {
+      newCardPopup.renderLoading(false);
+    });
+}
+// ------------- delete card function ---------------------
+function handleCardDelete(cardId) {
+  deleteCardPopup.open();
+  deleteCardPopup.setSubmitAction(() => {
+    deleteCardPopup.renderLoading(true);
+    api
+      .deleteCard(cardId)
```

Comment:
Needs correction: When I try to remove the second card, I see only text 'Yes' in the center of the screen, I don't see the popup